src/test/resources/testNG.xml

```diff
@@ -7,14 +7,14 @@
     <test name="Test Chrome">
         <parameter name="browser" value="Chrome"/>
         <classes>
-            <class name="tests.TC01_LoginWithValidAccount"/>
```

Comment:
why don't we keep both TC1 and TC2?

Reply:
I have just deleted class TC01 in XML when resolved conflict online. 
I updated new code into TC02 branch. Please check and review for me. Thank you so much!